common/OpTestHMC.py

```diff
@@ -55,6 +55,42 @@
 BOOTTIME = 500
 STALLTIME = 5
 
+class OpSecureBootUtilities():
+    '''
+    This class addresses the secureboot related pre-requisites.
+    '''
+
+    def __init__(self, console=None, distro=None):
+        self.console = console
+        self.distro = distro
+
+    def check_kernel_signature(self):
+        vmlinux = "vmlinuz"  # RHEL
+        if self.distro == 'SLES':
+            vmlinux = "vmlinux"
+
+        # Checking whether the kernel is signed or not
+        cmd = "strings /boot/%s-$(uname -r) | tail -1" % vmlinux
+        output = self.console.run_command(cmd)
+        if "Module signature appended" in output[0]:
+            kernel_signature = "signed_kernel"
+        else:
+            kernel_signature = "unsigned_kernel"
+        return kernel_signature
+
+    def check_os_level_secureboot_state(self):
+        '''
+        Check whether the secure-boot is enabled at os level.
+        To do this, check the entry of "00000002" in "/proc/device-tree/ibm,secure-boot" file.
+        If found, then secure-boot is enabled.
+        '''
+        os_level_secureboot = False
+        cmd = "lsprop /proc/device-tree/ibm,secure-boot"
+        output = self.console.run_command(cmd)
+        for line in output:
+            if '00000002' in line:
+                os_level_secureboot = True
+        return os_level_secureboot
```

Comment:
I think `os_level_secureboot` param is not required.

```
for line in output:
      if '00000002' in line:
            return True
 return False
```

Reply:
Done.